### ISSUE TITLE
Minor UI issues with latest master running locally

### ISSUE THREAD

**Issue by cpetzel** [statecharts/xstate-viz#8]
Thanks so much for this wonderful visualizer! It was great to use as we created our first machine, but it ended up growing in complexity, where we could not simply copy and paste the config into this online visualizer. 

We ended up forking it, and tried to configure this code to accept a machine as a prop (instead of relying on the json editor to recreate a machine every time). This works, but the rendering of the nodes and arrows do not look like the current version online...

This is a screenshot of our machine using your online visualizer...
![screen shot 2019-02-13 at 5 53 45 pm](https://user-images.githubusercontent.com/2036206/52757306-189e6600-2fb9-11e9-8efa-4d32ab89598f.png)

Here is a screenshot using our forked visualizer running locally. 
- The arrows are jagged, and are very hard to follow. 
- For `Transient` events, it renders a `?`
- For `Transient` events, it fails to draw the origin point for the arrow
- In some cases, some actions on the non-active statenode are still `active` when they should not be

![screen shot 2019-02-13 at 6 00 19 pm](https://user-images.githubusercontent.com/2036206/52757369-45527d80-2fb9-11e9-8cec-394bb268161c.png)

Is this repo missing a few enhancements compared to what is live already? I'd like to be able to build and run this locally, and these issues are the last remaining things. 

Thanks again!

**Comment by m-oscartong** [comment]
I also encounter similar issue, the transition lines between states are no longer curves and the connector dots becomes very huge that cover the whole page, which looks very different from the online visualizer 🤔. I end up grabbing the source code from the online visualizer and continue my work on top of that 😂

**Comment by davidkpiano** [comment]
Ah yeah, I was changing the algorithm for calculating the paths, and I guess I didn't change it back 😅 Still working on this.

**Comment by m-oscartong** [comment]
Any update on this issue? 🤔

**Comment by davidkpiano** [comment]
Try using the `gh-pages` branch.

This repo is mainly meant to host the current visualizer - the new one is being created in a private repo.

**Comment by davidkpiano** [comment]
Fixed in `master`
